security.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: markdown
+---
+## Security
+
+Best Practice: Reporting a Security Issue in Erlang/OTP
+
+## TLDR
+Please **do not create a public issue** for a security issue, use instead [https://github.com/erlang/otp/security](https://github.com/erlang/otp/security).
+
+Please follow this document in order to report security vulnerabilities in Erlang/OTP. 
```

Comment:
```suggestion
## Summary
Please create a new [Security Advisory](https://github.com/erlang/otp/security/advisories) for security issues. **Do not create a public github issue**.

Please follow this document in order to report security vulnerabilities in Erlang/OTP. 
```

Having a toddler at home has tought me that you should start with saying what you want someone to do, people always seem to miss the *not*.